docs/resources.md

```diff
@@ -0,0 +1,97 @@
+# Zero components
+**s3 bucket: Terraform-state:** 
+`<project>-<env>-terraform-state`
+Terraform state s3 bucket is used for storing your terraform state in a terraform [remote backend][tf-remote-backend]
+
+**dynamo db: Terraform statelock**:
+Terraform uses [state lock][tf-remote-state] to ensure actions that could contradict each others do not happen at the same time 
+
+**IAM user: `<project>-ci-user`**:
+This user is given the ECR full-access policy, our intention is you will push your container images to ECR via this IAM
+
+**Secret manager: `ci-user-aws-key`**:
+This is a credential key pair generated when creating an IAM user, we then store it in secret manager.
+
+**VPC:**
+Subnet for AWS, all our resources will be locally accessible with each others created in here
+by default everything will be under `10.10.xxx.xxx`, [see details][zero-subnet]
+
+**EKS-cluster**:
+Our main kubernetes stack, core of our infrastructure. Our business logic and application will be deployed here, Kubernetes enables ease of scaling / management of all our microservices. 
+Kubernetes will be high availability according to your [az settings][zero-az]
+
+**EKS: k8s-workers:**
+These are EC2 instances spawned according to your `ami-` image tag, any deployments and pods will be spun up in these EC2 instances. You can scale this up at a later time if your requirement increases
```

Comment:
Maybe de-emphasize the AMI. The user shouldn't really care about the AMI, as in the vast majority of cases they will just use the one provided by AWS.